docs/network-setup.md

```diff
@@ -96,6 +96,85 @@ your environment. For testing, you can add a public DNS server to
 nameserver 8.8.8.8
 ```
 
+## [Advanced] Setting Up a Bridge Interface
+
+<!-- markdownlint-disable-next-line MD024 -->
```

Comment:
What was the warning here ? Can we fix it instead of disabling it ?

Reply:
I don't see it failing in the [original failed CI build](https://buildkite.com/firecracker/firecracker-x86-64-amd-ci/builds/3166), but locally with markdownlint, the error that I was getting was related to using a duplicate heading because it would create links like `#on-the-host-1`

I can try to fix this this evening.

Reply:
The reason I chose to disable the lint instead of changing the header was to maintain the same section titling as the normal  --- i.e., non-bridge --- section.

Reply:
We are already disabling this rule [here](https://github.com/firecracker-microvm/firecracker/blob/main/tests/host_tools/mdl_style.rb#L6), so there is no need to do it here.